packages/iTwinUI-react/scripts/createComponent.js

```diff
@@ -81,7 +85,7 @@ const storiesFactory = (directory, name) => {
     template: `${copyrightHeader}
 import { Story, Meta } from '@storybook/react';
 import React from 'react';
-import { ${name}, ${name}Props } from '../../src/core';
+import { ${name}, ${name}Props } from '../../../packages/iTwinUI-react/src/core';
```

Comment:
Here should be
`from '@itwin/itwinui-react'`.
Check already existing stories and the import path.